integration-tests/md/extend.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python
+
+# requirements: md raid /dev/md0 with partitions sdb[1-4] and unused partition sdb5
```

Comment:
Suppose I learn how to set up a proper environment for this test.
I still have no idea how to tell if the test passes or not. If it crashes, it failed. But beyond that?